cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -367,7 +367,7 @@ DO: Use TLS 1.2 for your entire site. Get a free certificate [LetsEncrypt.org](h
 
 DO NOT: [Allow SSL, this is now obsolete](https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices#22-use-secure-protocols).
 
-DO: Have a strong TLS policy (see [SSL Best Practises](http://www.ssllabs.com/projects/best-practises/)), use TLS 1.2 wherever possible. Then check the configuration using [SSL Test](https://www.ssllabs.com/ssltest/) or [TestSSL](https://testssl.sh/).
+DO: Have a strong TLS policy (see [SSL Best Practises](https://www.ssllabs.com/projects/best-practices/index.html)), use TLS 1.2 wherever possible. Then check the configuration using [SSL Test](https://www.ssllabs.com/ssltest/) or [TestSSL](https://testssl.sh/).
```

Comment:
The first link contains no real content - it's mostly just a link to here:

https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices

Might as well just point directly there since we're updating it.

Reply:
Updated. @rbsec